documentation/source/overview/studies.rst

```diff
@@ -3,6 +3,24 @@
 Studies using SCT
```

Comment:
Oops! Sorry, I've accidentally based my branch off of the `-stitch` PR (I forgot to switch to master before creating the branch).

Give me one moment and I'll rebase onto master.

Reply:
There we are. :)